zod/visualization/lidar_3d.py

```diff
@@ -0,0 +1,233 @@
+                        # 8 vertices of a cube
+                        x=all_coords[0, :],
+                        y=all_coords[1, :],
+                        z=all_coords[2, :],
+                        i=i,
+                        j=j,
+                        k=k,
+                        name=class_name,
+                        legendgroup=class_name,
+                        legendgrouptitle=dict(text=class_name),
+                        showlegend=True,
+                        opacity=0.2,
+                        color=color,
+                        flatshading=True,
+                        contour=dict(color="white", show=True, width=10),
+                        alphahull=0,
+                    )
+                ]
+            )
+        else:
+            triangles = np.vstack((i, j, k)).T
+            x = all_coords[0, :]
+            y = all_coords[1, :]
+            z = all_coords[2, :]
+            vertices = np.vstack((x, y, z)).T
+            tri_points = vertices[triangles]
+            # extract the lists of x, y, z of the triangle vertices and connect them by a line
+            xe = []
+            ye = []
+            ze = []
+            for T in tri_points:
+                xe.extend([T[k % 3][0] for k in range(4)] + [None])
+                ye.extend([T[k % 3][1] for k in range(4)] + [None])
+                ze.extend([T[k % 3][2] for k in range(4)] + [None])
+
+            self.fig.add_traces(
+                [
+                    go.Scatter3d(
+                        x=xe,
+                        y=ye,
+                        z=ze,
+                        mode="lines",
+                        name="",
+                        line=dict(color=color, width=14),
+                    ),
+                    go.Scatter3d(
+                        x=xe,
+                        y=ye,
+                        z=ze,
+                        mode="lines",
+                        name="",
+                        line=dict(color="black", width=2),
+                    ),
+                ]
+            )
+
+    def show(self):
+        """Render graph."""
+        self.fig.update_layout(legend_title_text="Available Objects")
+        self.app.layout = dbc.Container(
+            [
+                get_header(self.app.title),
+                html.Hr(),
+                dbc.Row(dcc.Graph(id="my-graph", figure=self.fig), style={"flexGrow": "1"}),
+            ],
+            fluid=True,
+            style={"height": "100vh", "display": "flex", "flexDirection": "column"},
+        )
+
+        self.app.run_server(debug=False)
+
+
+if __name__ == "__main__":
+    #
+    # Zenseact Open Dataset
+    #
+    # NOTE! Set the dataset path
+    DATASET_ROOT = ""
+    VERSION = "full"  # "mini" or "full"
+
+    # initialize ZodFrames
+    zod_frames = ZodFrames(dataset_root=DATASET_ROOT, version=VERSION)
+
+    zod_frame = zod_frames[0]
+
+    # get the LiDAR point cloud
+    pcd = zod_frame.get_lidar_data()[0]
```

Comment:
should be 
`pcd = zod_frame.get_lidar()[0]`

Reply:
yes I was having difficulties since I have the PIP installed version and it is the same as on the docker image we use which is an old version where a lot of these functions had different names. I had problems trying to use the local version of the devkit without interfering with the installed one. I'll make these changes and try to test again since last time was quite a while ago